### PR TITLE
Recovery Password Using API

### LINKED ISSUE #31: Recovery password, using current email.

### PR DESCRIPTION
**Pull request by hearagmu**
#Recovery Password
In this PR we try to carry out a very simple mechanism to recover the password, in reality the API reset the user's password, sending it to the mail.

#Test
Try this PR by following the steps below
-Clone repository
-run terminal > npm install --force
-npx react-native start
-npx react-native run-android

#Screen GIF

2021-05-07_11h46_04

### CODE REVIEW

File: COVID19/src/utils/functions/services.js [hunk 1]
```diff
@@ -12,4 +12,16 @@ export default {
...
+        {
+          email: mail,
+        },
+        ).then((res) => {
+          return res.data.code;
+        }).catch((e) => {
+          return e.toString();
+        });
+    },
   };
```
Comment (fernandoamz) [inline COVID19/src/utils/functions/services.js:27]: new blank line here

**Review by fernandoamz — APPROVED** [review]
LGTM!

**Review by JorgeGarciaRa — APPROVED** [review]
Good job!

**Review by HRuvalcabaP — APPROVED** [review]
Good job!

**Pull request merged**